### PR TITLE
feat: Add FAQ component

### PR DESCRIPTION
**Pull request by Destiny-01**
# General Changes

- Adds FAQ feature
## Developer Notes

Added FAQ section with dummy details while I await the questions to be asked

---

## Author Checklist

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

## Reviewer Checklist

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  New third-party packages, if any, do not introduce potential security threats
- [ ]  There are no CI changes, or they have been OK’d by the devops team
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  There is at least one approval from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by Destiny-01** [comment]
#189

**Comment by drewcook** [comment]
@Destiny-01 This is failing the build in CI due to it not being linted.  There is a husky commit hook that is supposed to lint the files.  Please make sure the linted files are committed.

**Comment by drewcook** [comment]
@Destiny-01 I pushed some fixes to `main` that should help with this. Please pull the latest from main and merge it into your 
branch, then push back up to this PR.

File: components/Faq.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+				</AccordionDetails>
+			</Accordion>
+			<Accordion>
+				<AccordionSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel2a-content" id="panel2a-header">
+					<Typography>What's the idea behind music as NFT's?</Typography>
+				</AccordionSummary>
+				<AccordionDetails>
+					<Typography>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam incidunt cum officia nihil quisquam velit eligendi unde tempora autem accusamus alias ea quo distinctio ut quos eius, soluta earum aliquam accusantium explicabo! Quisquam ab, id consectetur unde impedit totam maxime.
+					</Typography>
```
Comment (drewcook) [inline components/Faq.tsx:24]: @noahdahlman Please provide text for general FAQs

**Comment by drewcook** [comment]
Closing in favor of #200

**Pull request closed without merging**